grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -144,6 +144,10 @@ class InvoiceItem implements Serializable {
 
     // Total shipment item value
     def getTotalItemPrice() {
+        if (invoice.invoiceType?.code == InvoiceTypeCode.PREPAYMENT_INVOICE) {
```

Comment:
Let's add a convenience method here as I suspect this will be a commonly repeated piece of code.

Invoice 
```
boolean isPrepaymentInvoice() { 
    return invoice.invoiceType?.code == InvoiceTypeCode.PREPAYMENT_INVOICE
}
```

Reply:
done